colossalai/kernel/triton/flash_decoding.py

```diff
@@ -261,6 +261,9 @@ def flash_decoding_attention(
     # NOTE use `triton.next_power_of_2` here to utilize the cache mechanism of triton
     # To optimize, revise batching/scheduling to batch 2^n sequences in a batch (preferred)
     grid = (triton.next_power_of_2(bsz), num_heads, triton.cdiv(triton.next_power_of_2(max_seq_len_in_batch), BLOCK_KV))
+    output = torch.empty((bsz, num_heads, head_dim), dtype=q.dtype, device=q.device) if output is None else output
+    grid_1 = (triton.next_power_of_2(bsz), num_heads)
```

Comment:
Why call it grid_1?

Reply:
ya, but I still do not know why the variable is named as `grid_1`, `grid` and `grid_1` have trivial difference in terms of their naming.

Reply:
My previous idea was to calculate the required grid for each kernel before executing the Triton kernel.

Reply:
What I want to discuss is why we name it with `_1`, what meaning does `_1` carry?

Reply:
My previous intention was to distinguish between the names of the two grids, but the naming convention I used might not have been rigorous enough.

Reply:
Ok, do avoid casual naming for the sake of clarity.